tools/c7n_azure/tests/templates/provision.sh

```diff
@@ -22,14 +22,15 @@ for file in "$templateDirectory"/*.json; do
       az group create --name $rgName --location $resourceLocation
       if [[ "$filenameNoExtension" =~ "keyvault-no-policies" ]]; then
         az group deployment create --resource-group $rgName --template-file $file \
-            --parameters "tenantId=$azureTenantId"
+            --parameters "tenantId=$azureTenantId" --no-wait
       elif [[ "$filenameNoExtension" =~ "keyvault" ]]; then
         azureAdUserObjectId=$(az ad signed-in-user show --query objectId)
         azureAdUserObjectId=${azureAdUserObjectId//\"}
 
         az group deployment create --resource-group $rgName --template-file $file \
             --parameters "tenantId=$azureTenantId" \
-                         "userObjectId=$azureAdUserObjectId"
+                         "userObjectId=$azureAdUserObjectId" \
+                         --no-wait
```

Comment:
It looks like the next line is looking up the name of the deployed vault so this will probably break it? 

I know Kirill is rewriting this script anyway to support functional tests soon.

Reply:
Yeah, I will send a PR with updated script soon.